### PR TITLE
Fix main page slider display on all devices

### LINKED ISSUE #9614: [BOOM-5805] Image slider not working properly for all screen sizes
> This issue has been migrated from the Forge. Read the [original ticket here](http://forge.prestashop.com/browse/BOOM-5805).

- _**Reporter:**_ ram.chandra178
- _**Created at:**_ Thu, 21 Jun 2018 07:51:49 +0200

<p>In front Office homepage, image slider UI is not proper. </p>
        
* How to reproduce the issue ?

open front office homepage-&gt;See the slider
<br/>
(Please refer to the attachments)

### PR DESCRIPTION
**Pull request by CaptainYouz**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This PR fixes the display (on small devices) of the slider in the main page of FO theme.
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #9614
| How to test?  | Check the display on different browser and browser size.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10341)

### CODE REVIEW

**Comment by CaptainYouz** [comment]
All visual changes approved by @TristanLDD .

File: themes/classic/_dev/css/components/imageslider.scss [hunk 1]
```diff
@@ -73,3 +57,52 @@
...
+    }
+    .carousel-control {
+      .icon-prev, .icon-next {
+        i {
+          font-size: 2rem;
+        }
+      }
+    }
+  }
+}
```
Comment (PierreRambaud) [inline themes/classic/_dev/css/components/imageslider.scss]: Missing new line at the end of file.

Comment (CaptainYouz) [reply themes/classic/_dev/css/components/imageslider.scss]: @PierreRambaud added in https://github.com/PrestaShop/PrestaShop/pull/10341/commits/15e7d09021d3e8e218b04fbc6cf60496cb82fd4d

**Review by eternoendless — APPROVED** [review]
Rebased

**Comment by PierreRambaud** [comment]
Thanks everyone ;)

**Pull request merged**